### PR TITLE
Fixed wrong worker count when client crashed

### PR DESCRIPTION
**Pull request by Luzifer**
@jbenet PTAL

### CODE REVIEW

**Comment by Luzifer** [comment]
(Look at https://github.com/Luzifer/gobuilder/pull/18/files?w=1 to ignore whitespace changes)

File: cmd/starter/main.go [hunk 1]
```diff
@@ -76,98 +77,112 @@ func main() {
 	}, docker.AuthConfiguration{})
 	orFail(err)
 
-	redisClient.Incr("active-workers")
-	defer func() {
-		redisClient.Decr("active-workers")
-	}()
+	c := cron.New()
+	c.AddFunc("0 */5 * * * *", announceActiveWorker)
+	c.Start()
```
Comment (jbenet) [inline cmd/starter/main.go:82]: is this removed when the process exits somehow? or is this added every time this runs?

**Comment by jbenet** [comment]
LGTM otherwise. thanks for `?w=1` link

File: cmd/starter/main.go (hunk 1 above)
Comment (Luzifer) [inline cmd/starter/main.go:82]: It doesn't get removed currently… But as the set is capable of sorting and I query only the last 10 minutes the clients will expire itself. I'll add a set remove based on time to clean the redis database…

**Pull request merged**